content/token-program.md

```diff
@@ -114,6 +115,95 @@ Recently, the first option was done away with and it became a requirement to dep
 
 In this case, we're creating a new account for a token mint so we use `getMinimumBalanceForRentExemptMint` from the `@solana/spl-token` library. However, this concept applies to all accounts and you can use the more generic `getMinimumBalanceForRentExemption` method on `Connection` for other accounts you may need to create.
 
+## Mint Token Metadata
+When you create a token, you want to make sure that the token shows up in user's wallets with a name, ticker, and image. Solana uses the [Token Metadata Program from Metaplex](https://docs.metaplex.com/token-metadata/specification#token-standards) to achieve this.
+The metadata account address is derived from the mint account. The metadata field requires a JSON file to be populated with at least the following:
+
+```tsx
```

Comment:
Just `typescript`. I know we used `tsx` a lot in the past (it was to work around a bug that is now fixed) but just `typescript` is fine now.

Reply:
Also: please use two spaces per `CONTRIBUTING.md`. Thanks@!